jack-and-the-beans/buddies-ios
# Allow Activities to be edited.

To Test
- Make sure create activity still works as expected.
- Try editing each field of an Activity.
- Ensure validation works when editing an Activity.

Also includes design change for this now dual-purpose Create/Edit Activity view.
